Sources/is-access-granted/main.swift

```diff
@@ -0,0 +1,77 @@
+/* Executable that will return a JSON with the granted permissions:
+ *  {
+ * 		"isAccessibilityGranted": boolean
+ * 		"isScreenRecordingGranted": boolean
+ *	}
+ *  
+ */
+
+ import AppKit
+
+func toJson<T>(_ data: T) throws -> String {
+	let json = try JSONSerialization.data(withJSONObject: data)
+	return String(data: json, encoding: .utf8)!
+}
+
+/**
+ *  Returns true if the Screen recording is granted by the user for this app. False
+ *  otherwise.
+ *  CREDITS: https://gist.github.com/soffes/da6ea98be4f56bc7b8e75079a5224b37
+ */
+func isScreenRecordingGranted() -> Bool {
```

Comment:
There's an easier check: https://github.com/karaggeorge/mac-screen-capture-permissions/blob/master/Sources/screen-capture-permissions/ScreenCapturePermissions.swift

Reply:
If I understood correctly, this one will prompt the dialog if there is no permission, which we are trying to avoid

Reply:
Ah, I didn't realize it. Too bad. I still cannot believe Apple hasn't provided proper APIs to check for status...

Reply:
They do for the Accessibility but couldn't find it for the Screen recording ...